spark-operator-docs/operator_probes.md

```diff
@@ -0,0 +1,82 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Operator Probes
+
+In Kubernetes world, the kubelet uses readiness probes to know when a container is ready to
+start accepting traffic, and it uses liveness probes to know when to restart a container. Here
+for Spark Operators, we provided those as below by default. You can override the values in
+values.yaml if you use Helm Chart to deploy the Spark Operator.
+
+```
+ports:
+- containerPort: 18080
+  name: probe-port
+livenessProbe:
+  httpGet:
+    port: probe-port
+    path: /healthz
+  initialDelaySeconds: 30
+  periodSeconds: 10
+readinessProbe:
+  httpGet:
+    port: probe-port
+    path: /startup
```

Comment:
Please follow K8s standard instead of `/startup`.
- https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health
> The Kubernetes API server provides 3 API endpoints (healthz, livez and readyz) to indicate the current status of the API server.

Reply:
updated